problem_builder/public/js/questionnaire.js

```diff
@@ -131,32 +131,22 @@ function MCQBlock(runtime, element) {
 
             display_message(result.message, messageView, options.checkmark);
 
-            var choiceInputs = $('.choice-selector input', element);
-            $.each(choiceInputs, function(index, choiceInput) {
-                var choiceInputDOM = $(choiceInput);
-                var choiceDOM = choiceInputDOM.closest('.choice');
-                var choiceResultDOM = $('.choice-result', choiceDOM);
-                var choiceTipsDOM = $('.choice-tips', choiceDOM);
-
-                if (result.status === "correct" && choiceInputDOM.val() === result.submission) {
-                    choiceDOM.addClass('correct');
-                    choiceResultDOM.addClass('checkmark-correct icon-ok fa-check');
-                }
-                else if (choiceInputDOM.val() === result.submission || _.isNull(result.submission)) {
-                    choiceDOM.addClass('incorrect');
-                    choiceResultDOM.addClass('checkmark-incorrect icon-exclamation fa-exclamation');
-                }
-
-                if (result.tips && choiceInputDOM.val() === result.submission) {
-                    mentoring.setContent(choiceTipsDOM, result.tips);
-                }
+            var choiceInputDOM = $('.choice-selector input[value="'+ result.submission +'"]');
+            var choiceDOM = choiceInputDOM.closest('.choice');
+            var choiceResultDOM = $('.choice-result', choiceDOM);
+            var choiceTipsDOM = $('.choice-tips', choiceDOM);
+            if (result.status === "correct"){
+                choiceInputDOM.addClass('correct');
+                choiceResultDOM.addClass('checkmark-correct icon-ok fa-check');
+            }else{
```

Comment:
Nit: Here, 3 lines above, and 5 lines below, there should be space before/after the `{` or `}`, for consistency with the rest of the file.